jota/src/test/java/org/iota/jota/IotaCoreApiTest.java

```diff
@@ -1,226 +1,298 @@
 
     private static final String TAG = "IOTA9TAG9999999999999999";
-    
+
+    // TODO this is online available to verify why the disabled test are not running
     private static IotaAPICore proxy;
 
+    @Mock
+    private IotaAPI iotaAPIMock;
+
+    // TODO this is online available to verify why the disabled test are not running
     @BeforeEach
     public void createProxyInstance() throws Exception {
         proxy = new IotaAPI.Builder().config(new FileConfig()).build();
     }
 
     @Test
-    @Tag("IntegrationTest")
-    public void shouldGetNodeInfo() throws ArgumentException {
-        GetNodeInfoResponse nodeInfo = proxy.getNodeInfo();
-        assertThat(nodeInfo.getAppVersion(), IsNull.notNullValue());
-        assertThat(nodeInfo.getAppName(), IsNull.notNullValue());
-        assertThat(nodeInfo.getJreVersion(), IsNull.notNullValue());
-        assertThat(nodeInfo.getJreAvailableProcessors(), IsNull.notNullValue());
-        assertThat(nodeInfo.getJreFreeMemory(), IsNull.notNullValue());
-        assertThat(nodeInfo.getJreMaxMemory(), IsNull.notNullValue());
-        assertThat(nodeInfo.getJreTotalMemory(), IsNull.notNullValue());
-        assertThat(nodeInfo.getLatestMilestone(), IsNull.notNullValue());
-        assertThat(nodeInfo.getLatestMilestoneIndex(), IsNull.notNullValue());
-        assertThat(nodeInfo.getLatestSolidSubtangleMilestone(), IsNull.notNullValue());
-        assertThat(nodeInfo.getLatestSolidSubtangleMilestoneIndex(), IsNull.notNullValue());
-        assertThat(nodeInfo.getNeighbors(), IsNull.notNullValue());
-        assertThat(nodeInfo.getPacketsQueueSize(), IsNull.notNullValue());
-        assertThat(nodeInfo.getTime(), IsNull.notNullValue());
-        assertThat(nodeInfo.getTips(), IsNull.notNullValue());
-        assertThat(nodeInfo.getTransactionsToRequest(), IsNull.notNullValue());
-    }
-
-    @Disabled
-    @Test //(expected = IllegalAccessError.class)
-    @Tag("IntegrationTest")
-    public void shouldGetNeighbors() throws ArgumentException {
+    public void shouldGetNodeInfo() {
+        GetNodeInfoResponse nodeInfoResponse = GetNodeInfoResponse.create("HORNET",
+                "0.5.3-rc3", 0, 0, "8", 0, 0,
+                "HSPDOTYNLKWIXYWLOCBVMGGKDQYBSC9BZJNJDEGXNIFMQNVJGZMYJNFVXBCVCKNJDOWKQDJXATSDUU999",
+                1942896,
+                "HSPDOTYNLKWIXYWLOCBVMGGKDQYBSC9BZJNJDEGXNIFMQNVJGZMYJNFVXBCVCKNJDOWKQDJXATSDUU999",
+                1942896, 1651535, 3, 0,
+                Instant.now().toEpochMilli(), 3, 0, new String[]{"RemotePOW", "WereAddressesSpentFrom"},
+                "GYISMBVRKSCEXXTUPBWTIHRCZIKIRPDYAHAYKMNTPZSCSDNADDWAEUNHKUERZCTVAYJCNFXGTNUH9OGTW");
+
+        when(iotaAPIMock.getNodeInfo()).thenReturn(nodeInfoResponse);
+
+        GetNodeInfoResponse nodeInfo = iotaAPIMock.getNodeInfo();
+
+        assertNotNull(nodeInfo.getAppVersion());
```

Comment:
Codacy found an issue: [JUnit assertions should include a message](https://app.codacy.com/manual/Iota-java/iota-java/pullRequest?prid=6319461)